tests/functional/Project.php

```diff
@@ -621,4 +621,44 @@ public function testGetActiveProjectIDsForGroup(): void
         // Check if a group with a project, assigned to a group project team, returns an empty array when $search_in_team is false
         $this->array(\Project::getActiveProjectIDsForGroup([$group->getID()], false))->isEmpty();
     }
+
+    public function testMyTasksURL()
+    {
+        $expected_criteria_params = http_build_query([
```

Comment:
I guess we could retrieve the expected value by calling `getMyTasksURL` directly.